packages/cli/src/cmds/update.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Result, FxError, err, ok } from "@microsoft/teamsfx-api";
+import path from "path";
+import { Argv } from "yargs";
+import activate from "../activate";
+import CliTelemetry, { makeEnvRelatedProperty } from "../telemetry/cliTelemetry";
+import {
+  TelemetryEvent,
+  TelemetryProperty,
+  TelemetrySuccess,
+} from "../telemetry/cliTelemetryEvents";
+import { getSystemInputs } from "../utils";
+import { YargsCommand } from "../yargsCommand";
+
+export class UpdateAadApp extends YargsCommand {
+  public readonly commandHead = "aad-app";
+  public readonly command = this.commandHead;
+  public readonly description = "Update the AAD App in the current application.";
+
+  public builder(yargs: Argv): Argv<any> {
+    this.params = {
+      folder: {
+        describe: "Select root folder of the project",
+        string: true,
+      },
+      env: {
+        describe: "Select an existing environment to udpate AAD App",
+        requiresArg: true,
+        string: true,
+      },
+    };
+    return yargs.options(this.params).version(false);
```

Comment:
Please don't add these parameters into the `this.params` because in this way `folder` and `env` will be auto ignored in the interactive mode.

```typescript
# this may be enough
yargs.options(EnvOptions).options(RootFolderOptions);
```